packages/plugin/src/theia.d.ts

```diff
@@ -9434,6 +9434,10 @@ declare module '@theia/plugin' {
          */
         appendLine(value: string): void;
     }
+    export enum DebugConsoleMode {
+        Separate = 0,
+        MergeWithParent = 1
+    }
```

Comment:
I would like to see additional documentation in here as well. Similar to vscode:

For `Separate`:

```ts
/**
 * Debug session should have a separate debug console.
 */
```

For `MergeWithParent`:

```ts
/**
 * Debug session should share debug console with its parent session.
 * This value has no effect for sessions which do not have a parent session.
 */
```

Also would you mind adding a newline before the enum declaration?

Reply:
Thanks for reminding, corrected.